src/tests/system/tests/test_gdm_passkey.py

```diff
@@ -0,0 +1,422 @@
+"""
+SSSD Passwordless GDM Passkey Tests
+
+:requirement: Passwordless GDM
+"""
+
+from __future__ import annotations
+
+import time
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.roles.ldap import LDAP
+from sssd_test_framework.topology import KnownTopology
+
+
+def client_setup_for_passkey(client, provider: IPA | LDAP | GenericProvider, pin: str | int | None = None):
+    # Configure SSSD
+    client.authselect.select("sssd", ["with-mkhomedir"])
```

Comment:
We also need `client.authselect.select("sssd", ["with-switchable-auth"])`. For that to work correctly you'll need the latest version from authselect and gdm/gnome-shell/mutter that is available at https://copr.fedorainfracloud.org/coprs/ipedrosa/passwordles-gdm/